app/sensors/TextField.js

```diff
@@ -151,11 +151,13 @@ export default class TextField extends Component {
 			"rbc-placeholder-inactive": !this.props.placeholder
 		});
 
+		const customClass = this.props.className ? this.props.className : '';
+
 		return (
 			<div className={`rbc rbc-textfield col s12 col-xs-12 card thumbnail ${cx}`} style={this.props.componentStyle}>
```

Comment:
Hi @dhruvdutt, `className` prop **must** be attached at the **root level** of the component. 

Since we are using `classnames` package here, you can use it to compute the classes as follows:

```
const cx = classNames({
 	"rbc-title-active": this.props.title,
 	"rbc-title-inactive": !this.props.title,
 	"rbc-placeholder-active": this.props.placeholder,
  	"rbc-placeholder-inactive": !this.props.placeholder
  }, this.props.className);
```

which should work here as we are already appending `cx` at the root level (which may or may not be the case with every component).

In cases where `classnames` package is not used in the component, you can directly do:

```
 className={`rbc rbc-textfield col s12 col-xs-12 card thumbnail ${cx} ${this.props.className ? this.props.className : ""}`}
```